Makefile

```diff
@@ -4,6 +4,8 @@ OS := $(shell uname -s)
 
 CONFIG_FILE?=config-files/config.yaml
 
```

Comment:
```suggestion
Match the release "0.1.6"
```

Reply:
Changed version in [d8b86ec](https://github.com/yetanotherco/aligned_layer/pull/423/commits/d8b86ec5e8afd35967a186b33f781dff77b21d7a).

Reply:
Added to readme in [1ab8eed](https://github.com/yetanotherco/aligned_layer/pull/423/commits/1ab8eed756779d80707289513ef36f4bec77b48b).